### PR TITLE
fixing redirects and broken links for sign-your-own-saml-csr guide

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?**https://github.com/okta/okta-developer-docs/pull/2643 converted https://developer.okta.com/docs/guides/sign-your-own-saml-csr/ to a single page guide, but it didn't implement the redirects properly or check thoroughly for broken links. This PR fixes that.
- **Is this PR related to a Monolith release?** No.
- Redirects tested on staging 11/16/21. Redirects all work fine, and as expected. 

### Resolves:

* [OKTA-438722](https://oktainc.atlassian.net/browse/OKTA-438722)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md [hunk 1]
```diff
@@ -350,7 +341,7 @@ To share the certificate that you created across multiple apps, clone it with th
 
 To share the certificate that you created across multiple IdPs, clone it with the [IdPs API](#top) by using the key `id` that you generated.
 
-Be sure to clone the certificate to every app or IdP that you want to share it with. After it is cloned, you need to [update the key credential](/docs/guides/sign-your-own-saml-csr/update-the-key-credential/) for the target app.
+Be sure to clone the certificate to every app or IdP that you want to share it with. After it is cloned, you need to [update the key credential](#update-the-key-credential) for the target app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md:344]:
```suggestion
Ensure that you clone the certificate to every app or IdP that you want to share it with. After the certificate is cloned, you need to [update the key credential](#update-the-key-credential) for the target app.
```

File: packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md [hunk 2]
```diff
@@ -350,7 +341,7 @@ To share the certificate that you created across multiple apps, clone it with th
 
 To share the certificate that you created across multiple IdPs, clone it with the [IdPs API](#top) by using the key `id` that you generated.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md:342]: `[IdPs API](#top)` - this link doesn't seem appropriate?

File: packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md:344]: On line 352, remove the capital letters from the link: `[Publish a CSR with a certificate](#publish-a-csr-with-a-certificate)`

(sorry, I can't make this suggestion on the exact line in this review)

**Review by vanngo-okta — APPROVED** [review]
The redirects look good! 🚢 
I found a couple of nits, but approving to not hold you back.

**Comment by chrisdmills-okta** [comment]
Thanks @vanngo-okta ! Nits fixed.

**Pull request merged**